### PR TITLE
ready to release v1.5.0

### PR DESCRIPTION
**Pull request by thinkerou**
- [x] Add break-backward remind log

### CODE REVIEW

**Comment by matheus-meneses** [comment]
When next version will be realeased?

**Comment by kylesliu** [comment]
Very Urgently Expecting tag1.5

**Comment by appleboy** [comment]
@thinkerou maybe move PR or issue to 1.6 milestone? See https://github.com/gin-gonic/gin/milestone/10

**Comment by thinkerou** [comment]
@appleboy done, but still have one pull request, please review, thanks!

**Comment by thinkerou** [comment]
@appleboy all pull request have updated, and freeze the master until release v1.5.0

**Comment by dalu** [comment]
get on with it

**Comment by matheus-meneses** [comment]
Is it available?

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,48 @@
...
+- [FIX] Bump github.com/mattn/go-isatty library to support Risc-V [#2019](https://github.com/gin-gonic/gin/pull/2019)
+- [FIX] Improve logger code [#1926](https://github.com/gin-gonic/gin/pull/1926)
+- [FIX] Fixed JSONP format (added semicolon) [#2007](https://github.com/gin-gonic/gin/pull/2007)
+- [FIX] Support go1.13 [#2038](https://github.com/gin-gonic/gin/pull/2038)
+- [FIX] Upgrade validator version to v9 [#1015](https://github.com/gin-gonic/gin/pull/1015)
+- [NEW] Add DisallowUnknownFields() in gin.Context.BindJSON() [#2028](https://github.com/gin-gonic/gin/pull/2028)
+- [FIX] Refactor(context.go): GetPostFormMap [#2051](https://github.com/gin-gonic/gin/pull/2051)
+- [FIX] Adjust Render.Redirect test case [#2053](https://github.com/gin-gonic/gin/pull/2053)
+- [FIX] Make countParams more readable [#2052](https://github.com/gin-gonic/gin/pull/2052)
+- [FIX] Use url.URL.Query instead of parsing query [#2063](https://github.com/gin-gonic/gin/pull/2063)
```
[the remark this answers is not in the document]
Comment (thinkerou) [reply CHANGELOG.md]: use `REFACTOR` instead of FIX?

**Comment by thinkerou** [comment]
@Dominik-K thanks! updated!

**Comment by thinkerou** [comment]
@appleboy please review, thanks!

**Comment by fifsky** [comment]
When will it be released

**Comment by goldyfruit** [comment]
> When will it be released

When the guys will have time to review. 
Please keep in mind that maintaining Gin is not their full time job, they are doing that on the side.

**Comment by thinkerou** [comment]
Hi, all, v1.5.0 released!

**Pull request merged**